.appveyor.yml

```
@@ -7,6 +7,6 @@ build_script:
 - cmd: >-
     mkdir build &&
     cd build &&
-    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
-    cmake --build . --target install &&
-    ctest -C Debug --output-on-failure
+    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_FLAGS="/O2 /W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
```

```suggestion
    cmake c:\projects\source -G "Visual Studio 17 2022" -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/O2 /W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
```

Though the generator flag may not be required, I think it's better to be explicit here

---

that's the wrong flag we are using Visual Studio 2019 in this test

---

Ok, we can just replace the 2022 with 2019 it's a valid generator also: https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#visual-studio-generators